sdk/src/loom2d/Events/GameControllerEvent.ls

```diff
@@ -0,0 +1,58 @@
+package loom2d.events
+{
+    /** A GameControllerEvent is dispatched in response to user input through a game controller.
+     * 
+     *  To be notified of game controller events, add an event listener to the stage. Children
+     *  of the stage won't be notified of game controller input.
+     *  
+     *  @see loom2d.display.Stage
+     */  
+    public class GameControllerEvent extends Event
+    {
+        /** Event type for a button that was released. */
+        public static const BUTTON_UP:String = "buttonUp";
+        
+        /** Event type for a button that was pressed. */
+        public static const BUTTON_DOWN:String = "buttonDown";
+
+        /** Event type for an axis being moved. */
+        public static const AXIS_MOTION:String = "axisMoved";
+
+        public static const CONTROLLER_ADDED:String = "addedController";
+        public static const CONTROLLER_REMOVED:String = "removedController";
+
+        private var mControllerID:uint;
+        private var mButtonID:uint;
+        private var mAxisID:uint;
+        private var mAxisValue:Number;
+        
+        /** Creates a new KeyboardEvent. */
+        public function GameControllerEvent(type:String, controllerID:uint=0, buttonID:uint=0, 
+                                      axisID:uint=0, axisValue:int=0)
```

Comment:
It expects a value between 0 and 1. Looks like I forgot to change it to Number in the constructor since switching from SDL_Joystick to SDL_GameController.